Directory.Build.props

```diff
@@ -33,6 +33,13 @@
     <IsCIBuild Condition="'$(TF_BUILD)' == 'true'">true</IsCIBuild>
   </PropertyGroup>
 
+  <!-- Library target frameworks -->
+  <PropertyGroup>
+    <LibraryTargetFrameworks>netstandard2.0;net6.0;net8.0</LibraryTargetFrameworks>
```

Comment:
I tried adding net9 also, but it caused problems which require code changes. so I gave up